addons/web/static/tests/setup.js

```diff
@@ -241,6 +242,7 @@ function removeUnwantedAttrsFromTemplates(templates, attrs) {
 export async function setupTests() {
     QUnit.testStart(() => {
         checkGlobalObjectsIntegrity();
+        patchWebsocketWorkerWithCleanup();
```

Comment:
oh god. what a nightmare. are you really sure you need to do that here? the goal is to remove all that code in the future